website/cue/reference/remap/functions/parse_url.cue

```diff
@@ -13,6 +13,15 @@ remap: functions: parse_url: {
 			required:    true
 			type: ["string"]
 		},
+		{
+			name: "default_known_ports"
+			description: """
+				If true and the port number is not specified in the input URL string (or matches the default port for the scheme), it will be populated from well-known ports for the following schemes: `http`, `https`, `ws`, `wss`, and `ftp`.
```

Comment:
nit: might want to split this up over multiple lines (I expected cuefmt to complain about it, but it appears it doesn't).